terraform/pipeline/workspace.py

```diff
@@ -0,0 +1,46 @@
+"""Used to set the environment-related variables at runtime (rather than build
+time) so that all the necessary pipeline variables are available."""
+
+import os
+import sys
+
+REASON = os.environ["REASON"]
+# the name of the variable that Azure Pipelines uses for the source branch
+# depends on the type of run, so need to check both
+SOURCE = os.environ.get("OTHER_SOURCE") or os.environ["INDIVIDUAL_SOURCE"]
```

Comment:
Very minor, take it or leave it

```suggestion
SOURCE = os.environ.get("OTHER_SOURCE", os.environ["INDIVIDUAL_SOURCE"])
```

Reply:
Thanks, but I think it's clearer as written.